app/backend/src/couchers/servicers/requests.py

```diff
@@ -68,15 +68,16 @@ def CreateHostRequest(self, request, context):
             if not is_valid_date(request.from_date) or not is_valid_date(request.to_date):
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.INVALID_DATE)
 
-            if request.from_date < least_current_date():
+            # A > B
+            if least_current_date() > request.from_date:
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_FROM_BEFORE_TODAY)
 
-            if request.to_date < least_current_date():
-                context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_TO_BEFORE_TODAY)
-
+            # B >= C
             if request.from_date >= request.to_date:
                 context.abort(grpc.StatusCode.INVALID_ARGUMENT, errors.DATE_FROM_AFTER_TO)
 
+            # No need to check A > C
```

Comment:
Could use `least_current_date()` instead of A, `request.from_date` instead of B, etc, but doesn't matter much.